modules/engine/src/model/model.ts

```diff
@@ -519,6 +528,36 @@ export class Model {
     }
     return this.pipeline;
   }
+
+  /** Throttle draw call logging */
+  _lastLogTime = 0;
+  _logOpen = false;
+
+  _logDrawCallStart(): void {
+    // IF level is 4 or higher, log every frame.
+    const logDrawTimeout = log.level > 3 ? 0 : LOG_DRAW_TIMEOUT;
```

Comment:
I assume that higher number = more logging, across the board? And if someone wants the firehose, setting `log.level = Infinity` would also work? These would be good details for https://luma.gl/docs/developer-guide/debugging#logging, I've been using `log.level = 3` since that's what's mentioned there.